__tests__/app_test_.js

```diff
@@ -371,4 +371,87 @@ describe("POST Error Handling", () => {
         expect(msg).toBe("Bad request: Username does not exist");
       });
   });
+});
+
+
+describe("GET /GET /api/reviews (queries)", () => {
+  test("200: Responds with an reviews object sorted by title in descending order", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=title&order=desc")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSortedBy("title", { descending: true });
+      });
+  });
+  test("200: Responds with an reviews object sorted by designer in ascending order", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=designer&order=asc")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews).toBeSortedBy("designer", { ascending: true });
+      });
+  });
+  test("200: Responds with an reviews object filter by the category of social deduction", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
```

Comment:
you can't put spaces directly into URLS. Have a quick google on how we can navigate this